src/throttledStore.tsx

```diff
@@ -279,17 +280,19 @@ export const createThrottledStore = (
         observableNotify: onObservableNotify,
         resetPendingNotifications: resetAllPendingNotifications,
         hasPendingSubscribers: () => pendingBroadcastNotification,
-        deferSubscriberNotifications: async action => {
+        deferSubscriberNotifications: async (action, shouldClearMemoizedForState) => {
```

Comment:
I suggest changing to parameter to be `{shouldClearMemoizedForState}` as options object to allow forward compatibility.
Also, it might be misleading, since it would clear cache only on dispatch, maybe we should name it `shouldDispatchClearCache` or something like that

Reply:
We wanted this to be a temporary solution, so that after we make sure it does not hurt performance, we'll remove this option and do it by default. In that case, do you still want an options object?

regarding the name - changed :)